modules/ebi-metagenomics/dram/distill/main.nf

```diff
@@ -0,0 +1,73 @@
+process DRAM_DISTILL {
+    tag "${meta.id}"
+    label 'process_high'
+
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container 
+        ? 'https://depot.galaxyproject.org/singularity/dram:1.3.5--pyhdfd78af_0'
+        : 'quay.io/biocontainers/dram:1.3.5--pyhdfd78af_0'}"
+
+    containerOptions {
+        def arg = "--volume"
+        if (workflow.containerEngine == 'singularity' || workflow.containerEngine == 'apptainer') {
+            arg = "--bind"
+        }
+        def mounts = [
+            "${params.reference_dbs}/dram_dbs/:/data/",
+            "${params.reference_dbs}/dram_dbs/DRAM_CONFIG.json:/usr/local/lib/python3.10/site-packages/mag_annotator/CONFIG",
```

Comment:
I think it's better to have the reference_dbs as a an input.. because otherwise you are forcing anyone that want to use this module to use that parameter name (which is very generic) or to modify the module